python/ray/tune/schedulers/pbt.py

```diff
@@ -675,24 +715,27 @@ def _summarize_hyperparam_changes(
             return summary_str
         for param_name in old_params:
             old_val = old_params[param_name]
-            assert param_name in new_params and param_name in operations, (
-                "`old_params`, `new_params`, and `operations` "
-                f"must all contain the key: '{param_name}'\n"
+            assert param_name in new_params, (
+                "`old_params` and `new_params` "
+                f"must both contain the key: '{param_name}'\n"
                 f"old_params.keys() = {old_params.keys()}\n"
-                f"new_params.keys() = {new_params.keys()}\n"
-                f"operations.keys() = {operations.keys()}\n"
+                f"new_params.keys() = {new_params.keys()}"
             )
             new_val = new_params[param_name]
             summary_str += f"{prefix}{param_name} : "
             if isinstance(old_val, Dict):
                 # Handle nested hyperparameters by recursively summarizing
                 summary_str += "\n"
+                nested_operations = operations.get(param_name, {})
                 summary_str += self._summarize_hyperparam_changes(
-                    old_val, new_val, operations[param_name], prefix=prefix + " " * 4
+                    old_val, new_val, nested_operations, prefix=prefix + " " * 4
                 )
             else:
-                op = operations[param_name]
-                arrow = f"--- ({op}) -->"
+                op = operations.get(param_name, None)
+                if not op:
+                    arrow = "----->"
+                else:
+                    arrow = f"--- ({op}) -->"
                 summary_str += f"{old_val} {arrow} {new_val}\n"
         return summary_str
```

Comment:
It seems that we have done quite some usability improvement here.
Do you mind adding an example like what is the summary_str and how to interpret it in the doc string?